src/python/tools/db2graph/marius_db2graph.py

```diff
@@ -0,0 +1,709 @@
+                        help='Path to the config file')
+
+    parser.add_argument('--output_directory',
+                        metavar='output_directory',
+                        type=str,
+                        default="./",
+                        help='Directory to put output data and log file')
+    return parser
+
+def config_parser_fn(config_name):
+    """
+    Takes the input yaml config file's name (& relative path). Returns all the extracted data
+
+    :param config_name: file name (& relative path) for the YAML config file
+    :returns:
+        - db_server: string denoting database server (initial support only for mariadb)
+        - db_name: name of the database you need to pull from
+        - db_user: user name used to access the database
+        - db_password: password used to access the database
+        - db_host: hostname of the database
+        - entity_node_sql_queries: list of sql queries used to define entity nodes
+        - edge_entity_entity_sql_queries: list of sql queries to define edges of type entity nodes to entity nodes 
+            & the names of edges
+        - edge_entity_feature_values_sql_queries: list of sql queries to define edges of type entity node to feature 
+            values & also the names of edges
+    """
+    input_cfg = None
+    input_config_path = Path(config_name).absolute()
+    
+    input_cfg = OmegaConf.load(input_config_path)
+
+    # db_server used to distinguish between different databases
+    db_server = None
+    if "db_server" in input_cfg.keys():
+        db_server = input_cfg["db_server"]
+    else:
+        logging.error("ERROR: db_server is not defined")
+        exit(1)
+
+    # db_name is the name of the database to pull the data from
+    db_name = None
+    if "db_name" in input_cfg.keys():
+        db_name = input_cfg["db_name"]
+    else:
+        logging.error("ERROR: db_name is not defined")
+        exit(1)
+
+    # db_user is the user name used to access the database
+    db_user = None
+    if "db_user" in input_cfg.keys():
+        db_user = input_cfg["db_user"]
+    else:
+        logging.error("ERROR: db_user is not defined")
+    
+    # db_password is the password used to access the database
+    db_password = None
+    if "db_password" in input_cfg.keys():
+        db_password = input_cfg["db_password"]
+    else:
+        logging.error("ERROR: db_password is not defined")
+    
+    # db_host is the hostname of the database
+    db_host = None
+    if "db_host" in input_cfg.keys():
+        db_host = input_cfg["db_host"]
+    else:
+        logging.error("ERROR: db_host is not defined")
+
+    # generate_uuid is used to identify whether to generate the uuid using entity nodes or 
+    # use the table_name-col_name-value as the unique identifier
+    generate_uuid = None
+    if "generate_uuid" in input_cfg.keys():
+        generate_uuid = input_cfg["generate_uuid"]
+    else:
+        # default is assumed as true
+        generate_uuid = True
```

Comment:
This should be default False if we end up keeping this functionality

Reply:
Removed the functionality